app/views/spells/index.haml

```diff
@@ -0,0 +1,30 @@
+%p#notice= notice
+%h1 Spells
+%table
+  %thead
+    %tr
+      %th Name
+      %th Level
+      %th Range
+      %th School
+      %th Casting time
+      %th Duration
+      %th Description short
+      %th Description long
+      %th{colspan: "3"}
+  %tbody
+    - @spells.order(:level, :name).each do |spell|
```

Comment:
fine-ish, pls avoid sql in views. Create a named scope instead and use it in controller :)

Reply:
Done, thanks for pointing that out.